File: apps/problems/Broken-SubArray/boilerplate-full/function.cpp
#include <iostream>
    #include <fstream>
    #include <vector>
    #include <string>
    #include <sstream>
    #include <climits>
    
    ##USER_CODE_HERE##
    
    int main() {
      std::ifstream file("/dev/problems/Broken-SubArray/tests/inputs/##INPUT_FILE_INDEX##.txt");
      std::vector<std::string> lines;
      std::string line;
      while (std::getline(file, line)) lines.push_back(line);
    
      file.close();
      int size_A;
  std::istringstream(lines[0]) >> size_A;
  std::vector<int> A(size_A);
  if(!size_A==0) {
  	std::istringstream iss(lines[1]);
  	for (int i=0; i < size_arr; i++) iss >> arr[i];
  }
  int B;
  std::istringstream(lines[1]) >> B;
      int result = solve(A, B);
      std::cout << result << std::endl;
      return 0;
    }
    

File: apps/problems/Broken-SubArray/boilerplate-full/function.js
##USER_CODE_HERE##

            const input = require('fs').readFileSync('/dev/problems/Broken-SubArray/tests/inputs/##INPUT_FILE_INDEX##.txt', 'utf8').trim().split('\n').join(' ').split(' ');
            const size_A = parseInt(input.shift());
const A = input.splice(0, size_A).map(Number);
  const B = parseInt(input.shift());
            const result = solve(A, B);
            console.log(result);
                